pkg/configOperator/configOperator.go

```diff
@@ -39,7 +40,9 @@ func (op ConfigOperator) StartService() {
 	log.Print("Entering watch loop")
 
 	go op.startPollingUPS()
-	op.startKubeWatchLoop()
+	for {
+		op.startKubeWatchLoop()
```

Comment:
@darahayes could you please document that this method is blocking and we have it in a for loop because watch might expire/timeout?

Reply:
@aliok sounds like a good idea!